store/tikv/2pc.go

```diff
@@ -96,7 +97,7 @@ type twoPhaseCommitter struct {
 	// We use it to guarantee GC worker will not influence any active txn. The value
 	// should be less than GC life time.
 	maxTxnTimeUse uint64
-	detail        *execdetails.CommitDetails
+	detail        unsafe.Pointer
```

Comment:
Why change to `unsafe.Pointer`?

Reply:
in the previous, test was failed due to race condition in https://github.com/pingcap/tidb/pull/11757/files#diff-499c236856cd9ce3300d3f5ccde41a23R441, after this PR detail will be accessed in forked goroutine.